### DISCUSSION TITLE
Data Table 结合使用 Select 组件引发的问题

### DISCUSSION THREAD

**Discussion by Monster-hybrid** [tusen-ai/naive-ui#4754] [Q&A]
在 Data Table 中使用 Select 组件，我改如何区获取Select 中的值
我打算使用onchange获取，但我不知道如何传参
希望有大佬看到回答以下，你的帮助，我将感激不尽
局部代码如下：
NSelect,
		{
			//按钮样式
			style: {
				display: 'flex',
				textAline: 'center',
				justifyContent: 'space-around',
				width: '100px'
			},
				placeholder: tagKey,
				//动态数据
				options: studentOption,
				onchange: () => test(这里不知道填什么)
              }

**Comment by Sepush** [comment]
use `value` `onUpdateValue` prop

**Reply by Monster-hybrid** [reply]
I have already find the method to solve it: use ["onUpdate:value"]: (value) => setMonitor(value)
